mlrun/utils/helpers.py

```diff
@@ -1082,8 +1101,14 @@ def get_class(class_name, namespace=None):
     return class_object
 
 
-def get_function(function, namespace):
-    """return function callable object from function name string"""
+def get_function(function, namespaces, reload_modules: bool = False):
+    """return function callable object from function name string
```

Comment:
As a general note please use capital letters for comments and docstring even though it is not really enforced yet